src/libs/export.c

```diff
@@ -902,18 +1135,23 @@ void gui_init(dt_lib_module_t *self)
   dt_gui_add_help_link(self->widget, "export_selected.html#export_selected_usage");
 
   d->dimensions_type = dt_bauhaus_combobox_new(NULL);
-  dt_bauhaus_widget_set_label(d->dimensions_type, NULL, _("unit"));
-  gtk_widget_set_tooltip_text(d->dimensions_type, _("unit in which to input the image size"));
-  dt_bauhaus_combobox_add(d->dimensions_type, _("pixels (file size)"));
-  dt_bauhaus_combobox_add(d->dimensions_type, _("cm (print size)"));
-  dt_bauhaus_combobox_add(d->dimensions_type, _("in (print size)"));
+  dt_bauhaus_widget_set_label(d->dimensions_type, NULL, _("set size"));
+  gtk_widget_set_tooltip_text(d->dimensions_type, _("choose a method for setting the output size"));
+  dt_bauhaus_combobox_add(d->dimensions_type, _("in pixels (for file)"));
+  dt_bauhaus_combobox_add(d->dimensions_type, _("in cm (for print)"));
+  dt_bauhaus_combobox_add(d->dimensions_type, _("in inchs (for print)"));
+  dt_bauhaus_combobox_add(d->dimensions_type, _("by scale (for file)"));
   dt_bauhaus_combobox_set(d->dimensions_type, dt_conf_get_int(CONFIG_PREFIX "dimensions_type"));
 
   d->print_width = gtk_entry_new();
-  gtk_widget_set_tooltip_text(d->print_width, _("maximum output width\nset to 0 for no scaling"));
+  gtk_widget_set_tooltip_text(d->print_width, _("maximum output width limit.\n"
+                                                "click middle mouse button sets to 0\n"
+                                                " for cancel the constraint"));
```

Comment:
should be "to cancel..." but I would just drop this line. I would rephrase the line above:

"click middle mouse button to reset to 0"

Reply:
I tried to explain the meaning of this operation. Think don't need it?

Reply:
Yes I think it is not needed and not proper English.